Client/Frontend/TabContentsScripts/FormAutofillHelper/FormAutofillHelper.swift

```diff
@@ -34,8 +34,8 @@ class FormAutofillHelper: TabContentScript {
 
     // MARK: - Script Message Handler
 
-    func scriptMessageHandlerName() -> String? {
-        return "creditCardMessageHandler"
+    func scriptMessageHandlerNames() -> [String]? {
+        return ["addressMessageHandler", "creditCardMessageHandler"]
```

Comment:
Can we change the names to be more verbose ["addressFormMessageHandler", "creditCardFormMessageHandler"]

Reply:
Done in https://github.com/mozilla-mobile/firefox-ios/pull/17697/commits/016266748e04008328baacd1d6d619e0895c8835